docs/guide/introduction.md

```diff
@@ -97,4 +97,39 @@ The rest of the documentation will include many specific, practical examples of
 
 ## Content schemas
 
-Nulla vitae elit libero, a pharetra augue. Nullam quis risus eget urna mollis ornare vel eu leo. Maecenas sed diam eget risus varius blandit sit amet non magna. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+The content field schema is another key concept that most Apostrophe development will use. The "schema" here simply refers to definition of fields allowed in a particular context (e.g., a page, piece, or widget type). This is important so that the user interface knows what fields to show editors. It also tells both the UI and server how to validate data for that context.
```

Comment:
```suggestion
The content field schema is another key concept in Apostrophe. The "schema" simply refers the content fields that define a particular page, content type, or widget.```

Reply:
Until we're using "content type" universally we should use "piece type" for consistency.